app/controllers/articles_controller.rb

```diff
@@ -37,14 +37,12 @@ def show
 
   def destroy
     @article.destroy
-    flash[:notice] = "Usunieto artykul"
+    flash[:notice] = "Usunięto artykuł"
     redirect_to articles_path
```

Comment:
Do destroy we wszystkich controllerach. (Article, Comment, Message) a co w przypadku gdy nie zostanie usunięty ? Powinienes być gotowy obsłużyć taki wyjątek